src/transformers/models/oneformer/processing_oneformer.py

```diff
@@ -76,13 +114,43 @@ def _preprocess_text(self, text_list=None, max_length=77):
         token_inputs = torch.cat(token_inputs, dim=0)
         return token_inputs
 
-    def __call__(self, images=None, task_inputs=None, segmentation_maps=None, **kwargs):
+    @staticmethod
+    def _validate_input_types(
+        images: Optional[ImageInput] = None,
+        task_inputs: Optional[PreTokenizedInput] = None,
+    ) -> None:
+        if task_inputs is None:
+            raise ValueError("You have to specify the task_inputs. Found None.")
+        elif images is None:
+            raise ValueError("You have to specify the images. Found None.")
```

Comment:
this would be the only thing left and we can do something like if task_inputs ^ images: you have to specifiy one of `task_inputs` or `images`